lib/services/slack_command_service.ex

```diff
@@ -41,7 +41,13 @@ defmodule Aprb.Service.SlackCommandService do
             end
           end            
         end
-        ":+1: Unsubscribed from #{Enum.join(removed_topics, " ")}"
+        # remove nil from list
+        removed_topics = Enum.reject(removed_topics, fn(x) -> x == nil end)
+        if Enum.count(removed_topics) > 0 do
+          ":+1: Unsubscribed from #{Enum.join(Enum.map(removed_topics, fn(x) -> "_#{x}_" end), " ")}"
+        else
+          "Can't unsubscribe! You were not subscribed to _#{Enum.join(topic_names, " ")}_ or topic doesn't exist."
```

Comment:
Maybe there's a version of this error message that doesn't require the "...or...," like "Can't find a matching subscription!" (minor).